core/jvm/src/test/scala/org/scalacheck/ScalaCheckFrameworkSpecification.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * ScalaCheck
+ * Copyright (c) 2007-2021 Rickard Nilsson. All rights reserved.
+ * http://www.scalacheck.org
+ *
+ * This software is released under the terms of the Revised BSD License.
+ * There is NO WARRANTY. See the file LICENSE for the full text.
+ */
+
+package org.scalacheck
+
+import org.scalacheck.Prop.proved
+import sbt.testing.{Selector, SuiteSelector, TaskDef, TestSelector}
+
+object ScalaCheckFrameworkSpecification extends Properties("ScalaCheckFramework") {
+
+  private val firstProp = "ScalaCheckFrameworkHelper.first prop"
+  private val secondProp = "ScalaCheckFrameworkHelper.second prop"
+  private val thirdProp = "ScalaCheckFrameworkHelper.third prop"
+
+  property("all props with SuiteSelector") = {
+    getPropNamesForSelectors(List(new SuiteSelector)) == List(firstProp, secondProp, thirdProp)
+    getPropNamesForSelectors(List(new SuiteSelector, new TestSelector(firstProp))) == List(
+      firstProp,
+      secondProp,
+      thirdProp)
+    getPropNamesForSelectors(List(new SuiteSelector, new TestSelector("no matches"))) == List(
+      firstProp,
+      secondProp,
+      thirdProp)
```

Comment:
What is the purpose of all the `==` expressions here besides the last one?
It looks like only the last one is going to be converted to `Prop` (via `propBoolean`).
The results of all the other `==` expressions are going to be discarded.
The same question for the next `property` as well.

Reply:
You're right, this was a mistake on my end. I've changed the tests so that all properties are checked now.